src/components/ComponentIndexPage/index.js

```diff
@@ -58,43 +61,120 @@ function sortByNewest(a, b) {
   return dateA - dateB;
 }
 
+const filterLabels = [
+  {
+    title: 'Framework',
+    options: ['React', 'Angular', 'Vue', 'Vanilla'],
+  },
+  {
+    title: 'Design asset',
+    options: ['Sketch', 'Azure', 'Adobe XD', 'Figma'],
+  },
+  {
+    title: 'Availability',
+    options: ['Open Source', 'IBM Internal'],
+  },
+  {
+    title: 'Maintainer',
+    options: ['Cloud Data & AI', 'Cloud PAL', 'Watson Health', 'Watson IoT'],
+  },
+];
+
 function ComponentIndexPage() {
   const components = useComponentIndexData();
   const [activeSortOption, setActiveSortOption] = useState(initialSortOption);
+  const [selected, setSelected] = useState([]);
   const [searchValue, setSearchValue] = useState('');
   const [debouncedSearchValue] = useDebounce(searchValue, 300);
   const searchClient = useMemo(() => new Fuse(components, searchOptions), [
     components,
   ]);
 
+  function filterItems(items, filters) {
```

Comment:
I think this function can move to the outermost scope now, right? Since it doesn't depend on anything from the component that isn't passed in as a parameter? 👀

Reply:
Oops yes. updated